### ISSUE TITLE
Create namespace automatically if the user doesn't have a namesapce

### ISSUE THREAD

**Issue by khaledkbadr** [zero-os/0-orchestrator#1085]
Currently we require the user to create itsyouonline namespace beforehand installing the orchestrator. namespace can be generated from the clientID, clientSecret passed into the configuration, it'd be easier to the user if that step was done automatically in the configuration service.

**Comment by zaibon** [comment]
I agree that this could be done automatically. But this doesn't belong to the configuration service.
The detail of the IYO credential need to be passed during the creation of the object cluster. In this step we can create the namespace if needed.
